### PR TITLE
Add plugin for QuoteManagement->placeOrder()

### PR DESCRIPTION
**Pull request by malachy-mcconnell**

### CODE REVIEW

**Comment by malachy-mcconnell** [comment]
The 'do not merge' label is applied because currently this code is untested

File: Plugin/QuoteManagementPlugin.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+     * @return array
+     * @internal param \Magento\Framework\Api\SearchCriteriaInterface $searchCriteria
+     */
+    public function beforePlaceOrder(
+        \Magento\Quote\Api\CartManagementInterface $subject,
+        $cartId,
+        \Magento\Quote\Api\Data\PaymentInterface $paymentMethod = null
+    )
+    {
+        // @TODO (Malachy): Add 'If module is enabled'
```
Comment (sdelbosc) [inline Plugin/QuoteManagementPlugin.php]: Which module?

Comment (malachy-mcconnell) [reply Plugin/QuoteManagementPlugin.php]: This module, Acquia_CommerceManager

**Comment by sdelbosc** [comment]
@nikunjkotecha for info, this is going to need to be ported on our side for V2 to work

**Pull request merged**